### PR TITLE
Modify to use claims as information.

### PR DESCRIPTION
**Pull request by KotaTakahashi9320**
# Summary
Modify to use claims as information.
# Purpose
config.json is still there because the information in the claim is not used.
# Contents
Delete config.json
Modify to use claims as information.
Fix test
# Testing Methods Performed
CI

### CODE REVIEW

**Comment by KotaTakahashi9320** [comment]
@arukuka 
I have corrected the points you raised.
I would appreciate your confirmation.

**Review by arukuka — COMMENTED** [review]
@KotaTakahashi9320 Thank you!
I commented on the point of concern, so please check it.

File: packages/server/beaver_triple_service/jwt/types.go [hunk 1]
```diff
@@ -3,7 +3,7 @@ package jwt
 import "github.com/golang-jwt/jwt/v4"
 
 type Party struct {
-	Id      int    `json:"id"`
+	Id      uint32    `json:"id"`
 	Address string `json:"address"`
```
Comment (arukuka) [inline packages/server/beaver_triple_service/jwt/types.go:7]: It's a very small detail, but I would appreciate it if you could align the columns of the struct tags.

```suggestion
	Id      uint32 `json:"id"`
	Address string `json:"address"`
```

Comment (KotaTakahashi9320) [reply packages/server/beaver_triple_service/jwt/types.go]: Corrected with [a8f97e7](https://github.com/acompany-develop/QuickMPC/pull/122/commits/a8f97e7d47f513efa7b99aa3cad34ecfcca0185a)

File: packages/server/beaver_triple_service/cmd/jwt_generator/sample/sample.yml [hunk 2]
```diff
@@ -18,4 +18,4 @@ party_info:
   - id: 2
     address: 10.0.2.20
   - id: 3
-    address: 10.0.3.20
+    address: 10.0.3.20
```
[the remark this answers is not in the document]
Comment (KotaTakahashi9320) [reply packages/server/beaver_triple_service/cmd/jwt_generator/sample/sample.yml]: Corrected with [2dd3e55](https://github.com/acompany-develop/QuickMPC/pull/122/commits/2dd3e55b912e64d76b40297257e0c8493c666b2b)

File: packages/server/beaver_triple_service/server/engine_to_bts/e2b.go [hunk 3]
```diff
@@ -35,8 +33,24 @@ type server struct {
...
+	with_envoy, ok := os.LookupEnv("WITH_ENVOY")
+	if !ok {
+		return false, status.Error(codes.Internal, "with envoy is not provided")
+	}
+	return strconv.ParseBool(with_envoy)
+}
+
 // モック時に置き換わる関数
-var GetPartyIdFromIp = func(reqIpAddrAndPort string) (uint32, error) {
+var GetPartyIdFromIp = func(ctx context.Context, reqIpAddrAndPort string) (uint32, error) {
```
[the remark this answers is not in the document]
Comment (KotaTakahashi9320) [reply packages/server/beaver_triple_service/server/engine_to_bts/e2b.go]: @arukuka 
Thanks for the review!
Corrected with [36aeb67](https://github.com/acompany-develop/QuickMPC/pull/122/commits/36aeb67ccb467db0a4d6a1284246bec6f79c6f76).

I would appreciate your confirmation.

**Review by arukuka — APPROVED** [review]
Thank you! :bow:

**Pull request merged**